fhir_2_ada-r4/zibs2020/payload/0.0/all_zibs.xsl

```diff
@@ -26,31 +26,31 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     <xsl:import href="contextContactEpisodeOfCare.xsl"/>
     <xsl:import href="ext-CopyIndicator.xsl"/>
     <xsl:import href="ext-MedicationAgreementPeriodOfUseCondition.xsl"/>
-    <xsl:import href="ext-TimeInterval-period.xsl"/>
     <xsl:import href="ext-TimeInterval-Duration.xsl"/>
+    <xsl:import href="ext-TimeInterval-period.xsl"/>
     <xsl:import href="nl-core-AddressInformation.xsl"/>
     <xsl:import href="nl-core-AdministrationAgreement.xsl"/>
     <xsl:import href="nl-core-BodyHeight.xsl"/>
     <xsl:import href="nl-core-BodyWeight.xsl"/>
     <xsl:import href="nl-core-ContactInformation.xsl"/>
     <xsl:import href="nl-core-ContactPerson.xsl"/>
+    <xsl:import href="nl-core-DispenseRequest.xsl"/>
+    <xsl:import href="nl-core-ext-StopType.xsl"/>
```

Comment:
Why not just ext-StopType (see extensions above)?

Reply:
Good question but out of scope for this ticket. Created new ticket: MP-1189